<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mouvement</class>
 <widget class="QDialog" name="mouvement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mouvement d'agent</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Res.qrc">
    <normaloff>:/icones/Icones/mouvement.jpg</normaloff>:/icones/Icones/mouvement.jpg</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Affectation          </string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="agentEdit">
             <property name="placeholderText">
              <string>Agent</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="fonctionEdit">
             <property name="placeholderText">
              <string>Fonction</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="serviceEdit">
             <property name="placeholderText">
              <string>Service</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="affectationTableView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btApercu">
           <property name="text">
            <string>Aperçu</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/10-Search-icon.png</normaloff>:/icones/Icones/10-Search-icon.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btSupprimer">
           <property name="text">
            <string>Supprimer</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/14-Delete-icon.png</normaloff>:/icones/Icones/14-Delete-icon.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Nomination                </string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLineEdit" name="agenEdit">
             <property name="placeholderText">
              <string>Agent</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="directionEdit">
             <property name="placeholderText">
              <string>Direction</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="nominationTableView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QPushButton" name="btAper">
           <property name="text">
            <string>Aperçu</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/10-Search-icon.png</normaloff>:/icones/Icones/10-Search-icon.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btSupprime">
           <property name="text">
            <string>Supprimer</string>
           </property>
           <property name="icon">
            <iconset resource="../Res.qrc">
             <normaloff>:/icones/Icones/14-Delete-icon.png</normaloff>:/icones/Icones/14-Delete-icon.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btFermer">
       <property name="text">
        <string>Fermer</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/gnome_application_exit.png</normaloff>:/icones/Icones/gnome_application_exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btFermer</sender>
   <signal>clicked()</signal>
   <receiver>mouvement</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>737</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
